youtube_dl/extractor/mirror.py

```diff
@@ -0,0 +1,73 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+from ..utils import (
+    ExtractorError,
+)
+
+import re
+
+
+class MirrorIE(InfoExtractor):
+    _VALID_URL = (r'https?://(?:www\.)?(?:mirror\.co\.uk)/.*')
```

Comment:
The problem is that video id is from brightcove and is hardcoded in webpage, not in link. How I must handle it?